dvc/remote/azure.py

```diff
@@ -187,6 +187,9 @@ def download(
                 msg = "failed to download '{}/{}'".format(bucket, path)
                 logger.warning(msg)
             else:
+                if os.path.exists(to_info["path"]):
```

Comment:
Let's use `move()` from `dvc/utils` instead of
```
if os.path.exists(to_info["path"]):
    os.remove(to_info["path"])
os.rename(tmp_file, to_info["path"])
```

Reply:
:+1: